chia/wallet/wallet_state_manager.py

```diff
@@ -1149,11 +1149,14 @@ async def handle_clawback(
         if is_recipient is not None:
             spend_bundle = SpendBundle([coin_spend], G2Element())
             memos = compute_memos(spend_bundle)
+            spent_height: uint32 = uint32(0)
+            if coin_state.spent_height is not None:  # pragma: no cover
```

Comment:
Why should we not test this?

Reply:
I don't know how to simulate the nuke wallet DB. I extended the resync unit test but it doesn't trigger the change.

Reply:
added test

Reply:
can we remove the pragma now?

Reply:
Looks like there are still lines that need covered, but this particular case seems addressed.  Feel free to close this discussion.  I'll dismiss my requested changes.  Thanks.